.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -52,9 +52,9 @@ assignees: ''
 <!-- If applicable, add screenshots to help explain your problem. -->
 
 ### Installation method
-   - [ ] chectl <!--(please specify the options of chectl tool) -->
-   - [ ] che-operator <!--(version of che-operator) -->
```

Comment:
are you removing `che-operator` as an option?

Reply:
`che-operator` 
because it is actually chectl with installer operator and it is better to know the full chectl command than a checkbox with `che-operator`

btw. added checkbox for olm way to install Che

Reply:
Yes, that is what I meant: installation via olm was missing.